app/models/invoice.rb

```diff
@@ -5,4 +5,6 @@ class Invoice < ApplicationRecord
   has_many :items, through: :invoice_items
 
   enum status: ["completed", "cancelled", "in progress"]
+
+  validates :status, presence: true 
```

Comment:
I think enums have a more specific shouldamatcher than can be used for validations, allows checking of values too.

it { should define_enum_for(:status).with_values([completed: 0, cancelled: 1, in_progress: 2]) }

not 100% sure how the space is handled with the "in progress"

Reply:
Good callout! I'll look into it and add a little more robustness to the test. Thanks Sam!